src/react-components/ui-root.js

```diff
@@ -1037,11 +1051,7 @@ class UIRoot extends Component {
                     e.target.blur();
                   }
                 }}
-                placeholder={
-                  this.occupantCount() <= 1
-                    ? "Nobody is here yet..."
-                    : `Send message to ${this.occupantCount() - 1} other${this.occupantCount() - 1 > 1 ? "s" : ""}...`
-                }
+                placeholder={this.occupantCount() <= 1 ? "Nobody is here yet..." : messageEntryPlaceholder}
```

Comment:
this regresses the UX which informs the user of the number of others in the room in the placeholder. this message should likely say "Send message to X others (+ #channel)"

Reply:
(this text, along with the 'nobody is here yet' state, was put in since user feedback was it was confusing previously where there was no indication of who the messages were going to go to)

Reply:
I would address that sentiment by making the visual presentation of the box look more like a chat instead of a mystery text box, but if you like the other text then I will change it back.

Reply:
i found the old text to be nicer because it conveyed some presence info beyond 0 or 1, so i'd advocate to switching it back